### PR TITLE
support requests up to 2.22.0 version

### PR DESCRIPTION
**Pull request by GeyseR**
Signed-off-by: Sergey Fursov <geyser85@gmail.com>

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -9,15 +9,15 @@ dockerpty==0.4.1
 docopt==0.6.2
 enum34==1.1.6; python_version < '3.4'
 functools32==3.2.3.post2; python_version < '3.2'
-idna==2.5
```
Comment (ulyssessouza) [inline requirements.txt]: Why bumping `idna`?

**Comment by ijc** [comment]
@GeyseR does the bump of idna relate to bumping requests somehow or is this an unrelated change?

**Comment by GeyseR** [comment]
Hm, I don't think it's a required change for the upgrade. I'll rollback idna lib and update requests to 2.22.0

**Comment by ijc** [comment]
Thanks! If we want to bump idna we should at least do it in a separate commit, if not a separate PR.

**Comment by GeyseR** [comment]
Ready

**Comment by GeyseR** [comment]
> I have zero clue what that has to do with your change
same for me. maybe Jenkins is not working with the rewtitten history of the branch? What do you think about submitting a new PR to check this assumption?

**Comment by ijc** [comment]
@GeyseR I kicked it to have another go, but it failed again.

I don't think we need a fresh PR in the first instance, but would certainly be worth rebasing this one onto the current `master` and force pushing, maybe that will unconfuse something...

**Comment by GeyseR** [comment]
hey @ijc, I've just done a rebase and force push, and it seems that the new Jenkins build has passed the previous fail point

**Comment by ijc** [comment]
CI is green, thanks @GeyseR for your contribution → merging.

**Pull request merged**